packages/devtools_app/lib/src/screens/debugger/call_stack.dart

```diff
@@ -38,7 +42,10 @@ class _CallStackState extends State<CallStack>
           itemExtent: defaultListItemHeight,
           itemBuilder: (_, index) {
             final frame = stackFrames[index];
-            return _buildStackFrame(frame, frame == selectedFrame);
+            return _buildStackFrame(
+              frame,
+              frame == selectedFrame || frame == _clickedOnFrame,
```

Comment:
so that we don't have two frames highlighted, we could change this to:
`_clickedOnFrame != null ? frame == _clickedOnFrame : frame == selectedFrame`

Reply:
Good idea, done!